pallets/avn-finality-tracker/src/lib.rs

```diff
@@ -217,251 +219,256 @@ pub mod pallet {
             }
         }
     }
-}
```

Comment:
Would it be possible to avoid this change?
Keeping the ```impl<T: Config> Pallet<T> {``` block outside the pallet mode would be preferable and will minimise the diff changes.
Same for avn-proxy pallet.